src/state-store.js

```diff
@@ -52,9 +49,8 @@ class StateStore {
 
         request.onsuccess = (event) => {
           let result = event.target.result
-          if (result) {
-            // TODO: remove this when state will be serialized only via JSON.
-            resolve(result.isJSON ? JSON.parse(result.value) : result.value)
+          if (result && !result.isJSON) {
```

Comment:
Even if we haven't shipped this to beta/stable yet, I kept the `isJSON` conditional just to make sure that when rebuilding master there is no hard crash also for our users on the "dev channel".